src/shims/ApiCompatBaseline.netfx.netstandard20.txt

```diff
@@ -66,15 +64,6 @@ TypesMustExist : Type 'System.Reflection.MemberTypes' does not exist in the impl
 TypesMustExist : Type 'System.Reflection.ParameterModifier' does not exist in the implementation but it does exist in the contract.
```

Comment:
I didn't look at these in any deal I just refreshed from the build, but I agree those don't look right.